communities/usc-project-athena/community-impact-proposal.md

```diff
@@ -0,0 +1,53 @@
+# Summary of the Community's Needs
+  * Community demographics:
+    * Females at USC or in the Southern California area who are interested in coding and tech
+    * At our all female hackathon in April 2017, we hosted over 310 hackers!
+    * Most attendees were from colleges such as USC, UCSD, UCI and UCLA
+    * We had 40 high schoolers attend as well.
+  * Communication:
+    * The community stays in touch through the Facebook group and the slack.
+    * Email will be used for communication between the community and the organizing team.
+  * Culture:
+    * The community is heavily dominated by CS students, but there are also students who are interested in coding, or have a CS minor.
+  
+# Proposal
+  * Timing:
+    * We'd like to host another hackathon next spring, hopefully in February. 
+    * We would also like to host talks, workshops, and a women's conference in the fall semester.
+  * Community demographics:
+    * We're planning on increasing our community size, and we will use social media and campus ambassadors from other schools to help spread the word of our organization and events.
+  * Goals:
+    * People get involved on campus to set themselves up for success in the future (ie. grad school, work)
+    * The students are all hoping to complete their degree, and most want to work in industry.
+    * Regardless of major, they are passionate about developing and learning new things.
+    * Activities such as hackathons, group projects, and workshops can be implemented to set the community up for success.
+  * Communication:
+    * The community will stay in touch through our Facebook group, and the slack that was created for AthenaHacks 2017.
+# Proposal's Impact
+  * Target audience:
+    * Girls in college and high school who are interested in tech and coding! 
+  * Community needs:
+    * Addressing the gender gap in the tech world and the CS major. 
+    * Making learning code and tech more inviting and welcoming to girls.
+  * Resources:
+    * Resources for workshops, empowerment talks, technical writing workshops
+    * Resources for our hackathon next year
+    * Talk to tech companies to see if they're willing to sponsor/send speakers
+  * Media:
+    * Promote it on our social media accounts
```

Comment:
What does this promotion look like? How far ahead of a talk do you promote? What is the best time for a promotional tweet/post? What's the best content? etc